common/src/main/java/com/wynntils/core/objects/MessageContainer.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.objects;
+
+import com.wynntils.features.user.overlays.GameUpdateOverlayFeature;
+import com.wynntils.mc.render.TextRenderSetting;
+import com.wynntils.mc.render.TextRenderTask;
+
+public class MessageContainer {
+    TextRenderTask message;
+    long endTime;
+
+    public MessageContainer(String message) {
+        this.message = new TextRenderTask(message, TextRenderSetting.DEFAULT);
+        this.endTime = System.currentTimeMillis()
+                + (long) (GameUpdateOverlayFeature.getInstance().gameUpdateOverlay.messageTimeLimit * 1000);
```

Comment:
Not liking that functionality in `core` depends on a Feature... :-/ 

I'm not sure of the best way to solve this, but I think it seems reasonable that the default time limit should be a part of the MessageContainer class instead.

I assume you have it this way so that it can be a `@Config` in the Feature... It's a tricky one.

Reply:
... or maybe this class should be an inner class/helper class to GameUpdateOverlayFeature?

Reply:
Yes, this is a helper class, but I including it in the feature seemed to be a little overwhelming, and I also do not want to see `GameUpdateOverlayFeature.MessageContainer` every time we need to use it.